src/relentless/model/potential/potential.py

```diff
@@ -375,3 +375,111 @@ def save(self, filename):
         data = self.to_json()
         with open(filename, "w") as f:
             json.dump(data, f, sort_keys=True, indent=4)
+
+
+class BondedPotential(Potential):
+
+    def __init__(self, keys, params, container=None, name=None):
```

Comment:
Is this all the same as the base class defaults? If yes, you don't need to implement the `__init__` method since it will just pass through to the parent.